game/commandline.h

```diff
@@ -43,6 +43,7 @@ class CommandLine {
     bool                doForceG1()        const { return forceG1;      }
     bool                doForceG2()        const { return forceG2;      }
     bool                doForceG2NR()      const { return forceG2NR;    }
+    uint32_t            getFxaaPreset()    const { return fxaaPreset;   }
```

Comment:
Project doesn't use `get*` notation (except really old code, that hasn't been cleaned yet) - should be just `fxaaPreset()`
Names for private variable are not strict, can be `uint32_t fxaaPresetId = 0`